i18n(pt-BR): Add deploy/vercel.md translation (#1742)
withastro/docs: src/pages/pt-br/guides/deploy/vercel.md

---
title: Faça o deploy do seu site Astro na Vercel
description: Como fazer o deploy do seu site Astro para a web na Vercel.
layout: ~/layouts/DeployGuideLayout.astro
i18nReady: true
---

> Você pode usar a [Vercel](http://vercel.com/) para fazer o deploy de um site Astro em sua rede Edge global com zero configuração.

Reviewer: Como "edge" é um termo bastante utilizado na documentação, acho que faz sentido utilizá-lo como outras keywords sem letra maiúscula.
```suggestion
Você pode usar a [Vercel](http://vercel.com/) para fazer o deploy de um site Astro em sua rede edge global com zero configuração.
```

Este guia inclui instruções para fazer o deploy na Vercel através da interface do site ou da CLI da Vercel.

## Configuração do Projeto

Você pode fazer o deploy do seu projeto Astro na Vercel como um site estático ou como site renderizado no servidor (SSR).

### Site Estático

Seu projeto Astro é um site estático por padrão. Você não precisa de nenhuma configuração adicional para fazer o deploy de um site Astro estático na Vercel. 

:::note

> Atualmente há um problema da Vercel exibindo uma página 404 nos sites Astro. Até que isso seja corrigido, você pode adicionar o seguinte arquivo de configuração na raiz do seu projeto:

Reviewer: Nit: nos -> em
```suggestion
Atualmente há um problema da Vercel exibindo uma página 404 em sites Astro. Até que isso seja corrigido, você pode adicionar o seguinte arquivo de configuração na raiz do seu projeto:
```

```json title="vercel.json"
{
  "cleanUrls": true,
}
```
:::

### Adaptador para SSR

Para habilitar o SSR em seu projeto Astro e fazer o deploy na Vercel:

Adicione o [adaptador para Vercel](/pt-br/guides/integrations-guide/vercel/) para habilitar o SSR em seu projeto Astro com o comando `astro add` a seguir. Isso irá instalar o adaptador e fazer as alterações apropriadas em seu arquivo `astro.config.mjs` em apenas um passo.

```bash
npx astro add vercel
```

Se preferir instalar o adaptador manualmente, conclua os dois passos a seguir:

> 1. Instale o [adaptador @astrojs/vercel](/pt-br/guides/integrations-guide/vercel/) nas dependências do seu projeto usando seu gerenciador de pacotes preferido. Se você está utilizando o npm ou não tem certeza, execute em seu terminal:

Reviewer: Faltou os "`" que marcam parte do texto como código da mesma forma que está na versão inglesa.
```suggestion
1. Instale o [adaptador `@astrojs/vercel`](/pt-br/guides/integrations-guide/vercel/) nas dependências do seu projeto usando seu gerenciador de pacotes preferido. Se você está utilizando o npm ou não tem certeza, execute em seu terminal:
```

    ```bash
      npm install @astrojs/vercel
    ```

1. Adicione duas novas linhas ao seu arquivo de configuração do projeto `astro.config.mjs`.

    ```js title="astro.config.mjs" ins={2, 5-6}
    import { defineConfig } from 'astro/config';
    import vercel from '@astrojs/vercel/serverless';

    export default defineConfig({
      output: 'server',
      adapter: vercel(),
    });
    ```

## Como fazer o deploy

Você pode fazer o deploy na Vercel através da interface do site ou da CLI (interface de linha de comando) da Vercel. O processo é o mesmo para sites Astro estáticos e SSR.

### Deploy através da interface do site

1. Faça o push do seu código para o seu repositório Git online (GitHub, GitLab, BitBucket).
2. [Importe o seu projeto](https://vercel.com/new) na Vercel.

> 3. A Vercel vai detectar automaticamento o seu site Astro e vai definir as configuração corretas.

Reviewer: Aqui houve alguns erros de digitação.
```suggestion
3. A Vercel vai detectar automaticamente o seu site Astro e irá definir as configurações corretas.
```

4. O deploy de sua aplicação está feito! (ex. [astro.vercel.app](https://astro.vercel.app/))

Depois que seu projeto foi importado e o deploy foi feito, todos os pushs subsequentes para branches vão gerar [Deploys de Visualização](https://vercel.com/docs/concepts/deployments/environments#preview) e todas as alterações feitas na branch de produção (geralmente “main”) resultarão em um [Deploy de Produção](https://vercel.com/docs/concepts/deployments/environments#production).

📚 Saiba mais sobre a [Integração da Vercel com o Git](https://vercel.com/docs/concepts/git).

### Deploy pela CLI

1. Instale a [CLI da Vercel](https://vercel.com/cli) e execute `vercel` para fazer o deploy.

    ```bash
    npm i -g vercel
    vercel
    ```

> 2. A Vercel vai detectar automaticamento o seu Astro e irá definir as configuração corretas.

Reviewer: Correção dos mesmos erros de digitação apresentados acima.
```suggestion
2. A Vercel vai detectar automaticamente o seu site Astro e irá definir as configurações corretas.
```
